sample/build.gradle.kts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id("com.android.application")
+    kotlin("android")
+}
+
+android {
+    compileSdk = 31
+
+    defaultConfig {
+        applicationId = "com.google.modernstorage.sample"
+        minSdk = 21
+        targetSdk = 31
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
+            useSupportLibrary = true
+        }
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
+    }
+
+    buildFeatures {
+        compose = true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = compose.versions.current.get()
+    }
+
+    packagingOptions {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
+        }
+    }
+}
+
+dependencies {
+
+    implementation(androidx.corektx)
+    implementation(androidx.appcompat)
+    implementation(androidx.material)
+    implementation(compose.ui)
+    implementation(compose.material)
+    implementation(compose.icons)
+    implementation(compose.uitoolingpreview)
+    implementation(androidx.lifecycleruntimektx)
+    implementation(compose.activity)
+    implementation(compose.navigation)
+
+    implementation(project(":permissions"))
```

Comment:
Can we use the new BOM?

Reply:
Sure, but it can't be imported as a project from my testing 

It would need to be done something like this

```
implementation("com.google.modernstorage:modernstorage-bom:${project.properties["VERSION_NAME"]}")
implementation("com.google.modernstorage:modernstorage-permissions")
implementation("com.google.modernstorage:modernstorage-photopicker")
implementation("com.squareup.okio:okio) // or implementation("${libs.okio.get().module}")
implementation("com.google.modernstorage:modernstorage-storage")
```

Only problem I have with this is that `VERSION_NAME` can be a unreleased version. It could keep a specific version until sample is updated for the next specific version

Reply:
Oh I thought it would work, well I guess it's only in the docs in that case 🙂